org.rdkit.knime.testing/META-INF/MANIFEST.MF

```diff
@@ -2,7 +2,7 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: Testing
 Bundle-SymbolicName: org.rdkit.knime.testing;singleton:=true
-Bundle-Version: 3.4.0.qualifier
+Bundle-Version: 3.5.0.qualifier
 Fragment-Host: org.rdkit.knime.nodes;bundle-version="3.4.0"
```

Comment:
I would also change the dependency version for org.rdkit.knime.types to 3.5.0 - otherwise we may see inconsistencies in strange installs